src/components/dashboard/sidebar/styles/SidebarCard.scss

```diff
@@ -0,0 +1,5 @@
+.big-number {
```

Comment:
Just for my own sanity's sake, this is called `big-number` because the side card has a large number representing the number of fully discounted courses, correct? When I first read it, my brain went to "a myriad of discounted courses" 🤦

Reply:
Also very minute point, but looking at the card example you built off, I see a small shadow behind that card that isn't on yours. Maybe it was a decision I missed but it should theoretically be an easy styling to add, correct? `box-shadow:` IIRC

Reply:
Yeah, I kind of expect paragon to be a more of a design system than it is. So yeah, I should add the drop shadow and/or consult with Brandon.